ocaml/xapi/xapi.ml

```diff
@@ -1170,6 +1170,10 @@ let server_init () =
             , [Startup.OnThread]
             , Remote_requests.handle_requests
             )
+          ; ( "Remove local ISO SR"
+            , [Startup.OnThread]
+            , fun () -> Vm_sysprep.on_startup ~__context
+            )
```

Comment:
This should be moved after the block below, where the master connection is established if running on a pool member, and dbsync has run.